contents/docs/session-replay/sharing.mdx

```diff
@@ -0,0 +1,74 @@
+---
+title: Sharing or embedding recordings
+---
+
+Recordings are typically consumed from the **Session Replay** part of PostHog but in some cases you may want to share specific recordings with people outside of your PostHog organization. 
+
+
+<iframe allowfullscreen width="100%" height="450" frameborder="0" src="https://app.posthog.com/embedded/VDVn0WRlvJdBomoFyy2Xg0Di2T1pEg"></iframe>
+
+> Above is an embedded recording of posthog.com ... on posthog.com  - very meta.
+
+
+## Sharing
+
+### via the UI
+
+When viewing any recording you can press `Share` and from there you can configure the recording for **external sharing**. There are some configuration options for the iframe as well as the ability to specify that it should start at a particular timestamp.
+
+### via the API
+
+A more advanced use case may be automating this process, for example if you want to automatically embed particular recordings in some sort of support tool whenever a user reports a bug.
+
+1. Retrieve the `$session_id` of the recording you want to share.
+
+```js
+// Frontend - where the recording is taking place
+const sessionID = posthog.get_session_id()
+// You need to handle sending this to your backend
+sendToSupportTool(sessionID)
+```
+
+
+2. Once you have the `session_id`, in your backend you can use your [Personal API Key](/docs/api#personal-api-keys-recommended) to enable that recording for sharing
+
+```node
+// Node.JS backend example
+const projectID = 123 // Found in Project Settings
```

Comment:
```suggestion
const projectID = 123 // Found in [Project Settings](https://app.posthog.com/project/settings)
```

Reply:
I avoided this because of EU but I guess we can accept that situation